packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -112,6 +112,14 @@ export function resolvePluginConfig(
 		experimental: pluginConfig.experimental ?? {},
 	};
 	const root = userConfig.root ? path.resolve(userConfig.root) : process.cwd();
+	const buildTimeEnv = vite.loadEnv(viteEnv.mode, root, [
+		"CLOUDFLARE_",
+		"WRANGLER_",
```

Comment:
Do we need to support `WRANGLER_` here as well? I think once https://github.com/cloudflare/workers-sdk/pull/10652 is merged, we could just support `CLOUDFLARE_` seeing it's a new feature.

Reply:
e.g.
```ts
const cloudflareEnv = vite.loadEnv(viteEnv.mode, root, [
  "CLOUDFLARE_",
  "WRANGLER_HYPERDRIVE_LOCAL_CONNECTION_STRING_",
]);
```

Reply:
Updated in https://github.com/cloudflare/workers-sdk/pull/10677/commits/d2ab32d64ed45f194dff81f999e683e2e6797e45 if you are fine with my suggestion